sys/kern/src/syscalls.rs

```diff
@@ -753,15 +753,16 @@ fn irq_control(
 ) -> Result<NextTask, UserError> {
     let args = tasks[caller].save().as_irq_args();
 
-    let operation = match args.control {
+    let operation = match args.control & 1 {
```

Comment:
feel free to ignore me, but i would personally prefer to write these masks as 
```suggestion
    let operation = match args.control & 0b01 {
```
as it feels a bit more obvious which bit we are picking out

Reply:
Added a bitflags type and removed the constants.